From 29dee721b4c99c50a41f3c85b91f593b23ba3d7f Mon Sep 17 00:00:00 2001
From: Arpit Agarwal <arp@apache.org>
Date: Fri, 26 May 2017 16:02:40 -0700
Subject: [PATCH 2548/2748] HDFS-11891. DU#refresh should print the path of
 the directory when an exception is caught.
 Contributed by Chen Liang.

(cherry picked from commit 76c15121d8e0f170b537c478aaf9126a2733d4ed)

Change-Id: Ic8913f85273d15033d4745d1da4a84834d5ef11f
---
 .../src/main/java/org/apache/hadoop/fs/DU.java     |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/DU.java b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/DU.java
index b64a19d..6e374c9 100644
--- a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/DU.java
+++ b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/DU.java
@@ -52,7 +52,8 @@ protected synchronized void refresh() {
     try {
       duShell.startRefresh();
     } catch (IOException ioe) {
-      LOG.warn("Could not get disk usage information", ioe);
+      LOG.warn("Could not get disk usage information for path {}",
+          getDirPath(), ioe);
     }
   }
 
-- 
1.7.9.5

